src/main/kotlin/cellsim/simulate/Neighborhood.kt

```diff
@@ -0,0 +1,56 @@
+package cellsim.simulate
+
+import cellsim.misc.PairXY
+import cellsim.misc.checkRectCoordinates
+
+// Describes neighborhood of a cell, which is a set of cells relative to each cell
+// which can influence that cell ((0, 0) is always in neighborhood)
```

Comment:
```suggestion
/// Describes neighborhood of a cell, which is a set of cells relative to each cell
/// which can influence that cell ((0, 0) is always in neighborhood)
```

Komentarze stanowiące publiczną dokumentację piszemy zaczynając od ///